webview-ui/src/comp/Klusterlet.tsx

```diff
@@ -0,0 +1,42 @@
+import { OcmResource } from '../../../src/data/loader'
+import { ConditionTableComponent } from '../common/ConditionTable';
+import { DateFormat } from '../common/common';
+import { OcmLabels } from '../common/Labels';
+
+type klusterletProps = {
+    klusterlet: OcmResource[]
+}
+
+export default function ShowKlusterlet(Props: klusterletProps) {
+
+
+
+    const row = Props.klusterlet.map(klusterlet => {            
+        return klusterlet.kr.status.conditions.map( (condition:any) => { 
+            return [new Date(condition.lastTransitionTime).toLocaleString("en-US",DateFormat),
+                    condition.message,
+                    condition.reason,
+                    condition.status
+                ]      
+            })
+        })
+    return (
+        //TODO-Add panel 
+        //TODO-add Klusterlet dashboard 
```

Comment:
Are these TODO comments for future attention or for this PR?

Reply:
future PR, already exists on another branch